.npmrc

```diff
@@ -0,0 +1,2 @@
+package-lock = false
+@themuse:registry=https://verdaccio.inf.themuse.com/
```

Comment:
this is needed to get the dependencies from verdaccio, such as in `package.json`, `@themuse/conflator`

Reply:
Cool